module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -101,6 +101,16 @@ public function getTranslatorLocale($default = 'en')
      */
     public function translate($target, $tokens = [], $default = null)
     {
+        if ($this->getTranslatorLocale() == "keys") {
+            $keyValueToString = function ($key, $val) {
+                return "$key = $val";
+            };
+            $tokenDetails = empty($tokens)
+                ? ''
+                : '| [' . implode(', ', array_map($keyValueToString, array_keys($tokens), array_values($tokens))) . ']';
+            return "*$target$tokenDetails*";
```

Comment:
At this point in the code, $target might be an array (in situations where a text domain is represented as an array instead of using a `::` delimiter -- you can see this, for example, on the facet list on the search home screen in a default configuration). This causes the string to be displayed as `*Array*`. I think you might want to move all this logic below the extractTextDomain call so that you can guarantee that you have a string and not some other thing. You might also want to display the text domain when it's a value other than default.

Reply:
I didn't think about this. Fixed it.